buildSrc/src/main/kotlin/com/datadog/gradle/plugin/jsonschema/PokoGenerator.kt

```diff
@@ -153,6 +157,10 @@ class PokoGenerator(
                 EXTENSION_FUNCTIONS_PACKAGE_NAME,
                 TO_JSON_ELEMENT_EXTENSION_FUNCTION
             )
+            fileBuilder.addImport(
+                EXTENSION_FUNCTIONS_PACKAGE_NAME,
+                FROM_JSON_ELEMENT_EXTENSION_FUNCTION
```

Comment:
I do not see where exactly are you using this method in the `PokoGeneratorDeserializer`